pkg/ruler/notifier.go

```diff
@@ -26,11 +26,13 @@ import (
 )
 
 type NotifierConfig struct {
-	TLS       tls.ClientConfig `yaml:",inline"`
-	BasicAuth util.BasicAuth   `yaml:",inline"`
+	TLSEnabled bool             `yaml:"tls_enabled" category:"advanced"`
+	TLS        tls.ClientConfig `yaml:",inline"`
+	BasicAuth  util.BasicAuth   `yaml:",inline"`
 }
 
 func (cfg *NotifierConfig) RegisterFlags(f *flag.FlagSet) {
+	f.BoolVar(&cfg.TLSEnabled, "ruler.tls-enabled", true, "Enable TLS for gRPC client connecting to alertmanager-client.")
```

Comment:
"client" is repeating
```suggestion
	f.BoolVar(&cfg.TLSEnabled, "ruler.tls-enabled", true, "Enable TLS for gRPC client connecting to alertmanager.")
```

Reply:
Addressed! Thanks!